### PR TITLE
feat: add new api notice

### PR DESCRIPTION
**Pull request by PapaZouk**
## Description

- Add a notice to the documentation page about upcoming API changes.
- Removes links to the smart surveys form.

Related issue: [BL-16184](https://dvsa.atlassian.net/browse/BL-16184)

Screenshot 2024-03-28 at 15 27 03

## Before submitting (or marking as "ready for review")

- [x] Does the pull request title follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) specification?
- [ ] Have you performed a self-review of the code
- [ ] Have you have added tests that prove the fix or feature is effective and working
- [x] Did you make sure to update any documentation relating to this change?

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -31,6 +31,10 @@
     </p>
     <h1 class="heading-xlarge u-space-b15">MOT history API</h1>
 
+    <div class="u-margin-top-50">
+        <p class="heading-large u-space-b10">A new version of the MOT History API will be available within the next X weeks.  Sign up for it <a href="https://submit.forms.service.gov.uk/form/2382/register-for-mot-history-api/7519" target="_blank">here</a>.</p>
```
Comment (ccurran01) [inline docs/index.html]: Let's get Ben to check this content and confirm if this link should go to the register page of the documentation or straight to the registration form

File: docs/index.html [hunk 2]
```diff
@@ -537,8 +540,8 @@ <h2 class="heading-medium">Change log</h2>
 
             <h2 class="heading-medium">Support</h2>
 
-            <p>If you have any issues using MOT history API, complete <a href="https://www.smartsurvey.co.uk/s/MOT_History_TradeAPI_Access_and_Support/" target="_blank">
```
Comment (ccurran01) [inline docs/index.html:540]: Keep the support links

File: docs/index.html [hunk 3]
```diff
@@ -62,8 +66,7 @@ <h2 class="heading-medium">Overview</h2>
 
             <h2 class="heading-medium">Apply for API access</h2>
```
Comment (ccurran01) [inline docs/index.html]: Remove this whole section

File: docs/index.html (hunk 1 above)
Comment (ccurran01) [reply docs/index.html]: Content per screenshot in ticket

File: docs/index.html [hunk 4]
```diff
@@ -662,9 +665,7 @@ <h2 class="heading-medium">Response codes</h2>
 
             <h2 class="heading-medium">Support</h2>
 
-            <p>If any issues are experienced using MOT History API service, users are requested to use the following
```
Comment (ccurran01) [inline docs/index.html:665]: Keep this link too

File: docs/index.html [hunk 5]
```diff
@@ -557,8 +560,8 @@ <h2 class="heading-medium">Introduction</h2>
 
             <h2 class="heading-medium">Apply for API access</h2>
```
Comment (ccurran01) [inline docs/index.html]: Remove this whole section

File: docs/index.html (hunk 1 above)
Comment (PapaZouk) [reply docs/index.html]: Updated

**Review by ccurran01 — APPROVED** [review]

**Review by faggc — APPROVED** [review]

**Pull request merged**